dev_tools/rubocop/base.yml

```diff
@@ -0,0 +1,4 @@
+AllCops:
+  Exclude:
+    - 'bin/{rails,rake}'
+    - 'db/**/*'
```

Comment:
Does this exclude `db/seeds.rb`? Sometimes that can get quite complex so I'm not sure we'd want to exclude it.

Reply:
I changed it to be more explicit.